apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -19,6 +19,8 @@ import {isMobileDevice} from '@cdo/apps/util/browser-detector';
 
 const MAX_SEARCH_RESULTS = 40;
 const MAX_HEIGHT = 460;
+const MODAL_HEADER_HEIGHT = 95;
+const MODAL_FOOTER_HEIGHT = 50;
```

Comment:
where do these height constants come from?

Reply:
I added more comments - they come from the title, search bar, and categories. Unfortunately, I'm not sure if there is a better way to decode this.